sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -485,10 +552,9 @@ def test_assert_compatible_argkmin_results():
             np.array([[2, 1, 4, 5, 3]]),
             **tols,
         )
-
-
+        
 @pytest.mark.parametrize("check_sorted", [True, False])
-def test_assert_compatible_radius_results(check_sorted):
+def test_assert_compatible_radius_results(check_sorted: bool):
```

Comment:
scikit-learn does not use type annotations at all. Can you remove all of them? Thank you.